src/handlers/createProduct.ts

```diff
@@ -0,0 +1,33 @@
+import { v4 as uuidv4 } from "uuid";
+
+import {HEADERS as headers, STATUS_CODES} from "../../constants";
+import {DbConnector} from "../data-layer/db-connector";
+import {APIGatewayProxyResult} from "aws-lambda";
+
+const validateProductCreationInput = (description, count, title, price) => {
+    const areAllArgumentsPresent = description && count && title && price;
+    if (!areAllArgumentsPresent) throw Error('Not all required arguments were provided (title, description, price, count)');
+};
+
+export const main = async (event): Promise<APIGatewayProxyResult> => {
+    try {
+        const { description, count, title, price } = JSON.parse(event.body);
+        const id = uuidv4();
+        validateProductCreationInput(description, count, title, price);
+        const productId = await DbConnector.createProduct({description, count, title, price, id});
+
+        return {
+            statusCode: STATUS_CODES.SUCCSESS,
+            headers,
+            body: productId,
```

Comment:
A product is created, but there is a message "can't parse JSON.  Raw result:" in the body of the response instead of an product id
![image](https://user-images.githubusercontent.com/103493950/222449366-ae283a16-a145-4836-b1a0-dcf92da564d1.png)

Reply:
This is swagger limitation. The structure of the response looks like
```
response: {
  body: 'the_id_of_created_product'
}
```
And swager tries to perform `JSON.parse(response.body)`, but it is ridiculous.